src/aiida/cmdline/commands/cmd_code.py

```diff
@@ -236,8 +236,15 @@ def show(code):
 @verdi_code.command()
 @arguments.CODE()
 @arguments.OUTPUT_FILE(type=click.Path(exists=False))
+@click.option(
+    '-u',
+    '--unsorted',
+    is_flag=True,
+    default=True,
+    help="Don't alphabetically sort output yaml configuration file.",
+)
```

Comment:
Are you sure this works? The default seems to be `True` so it will always be unsorted and there is no way to change the value. You would be better off to use a switch in this case rather than a flag. I would suggest
```suggestion
@click.option(
    '--sort/--no-sort',
    default=False,
    help="Sort the keys of the output YAML.",
)
```

Reply:
>  The default being True means it's unsorted by default, leading to sort_keys=True

That's not true though, right? If the default is `True`, then `sort_keys=True` as you say, but then the output will be _sorted_. I think it is kind of confusing to have it defined this way. If `unsorted` is `True` than you are _sorting_ 🤔 

The downside of this approach is still that it will be difficult to change the default. Imagine that we would want to change the behavior in the future to make the output unsorted by default, then you would either have to change the option name or force users to specify `--unsorted` if they want the output sorted 😅 

The standard practice for these kinds of switches is to use the `--option/--no-option` switch. This will allow changing the default without having to change the options and it is a lot clearer as to what is happening.

> Though, using the --sort/--no-sort option in that case changes the default to being unsorted, which I wanted to avoid as people might be used to the output being sorted by default as that was the previous behavior.

You can still set the default value to be sorted, i.e. do:
```python
@click.option('--sort/--no-sort', is_flag=True, default=True)
def main(sort):
```

Reply:
Yeah, you're right. Not sure why it seemed fine, but I just checked again, and the behavior is reverse, as expected from the actual code... :upside_down_face: The advantage of the `--option/--no-option` switch makes sense, already seeing the confusion that was emerging now on my side. I'll update it with the `--sort/--no-sort` switch, then it should be ready to go. Thanks!